src/lib/rooibos/RooibosSessionInfo.ts

```diff
@@ -78,7 +78,7 @@ export class SessionInfo {
             }
             //'testSuite  ' + testSuite.name);
             for (let testGroup of testSuite.getTestGroups()) {
-                testGroup.isIgnored = testGroup.isIgnored || testSuite.isIgnored || this.isExcludedByTag(testGroup, true);
+                testGroup.isIgnored ||= testSuite.isIgnored || this.isExcludedByTag(testGroup, true);
```

Comment:
I'm not sure I like this specific rule. `??=` is clear (and familiar) to me. But `||=` is not. I prefer the original code to this for boolean expressions.

Reply:
This applies to all instances of `||=` across the project, not just this location.